pwiz_tools/Skyline/Executables/SkylineBatch/SkylineBatch/SkylineBatchConfigForm.cs

```diff
@@ -146,7 +147,14 @@ private void btnSkylineFilePath_Click(object sender, EventArgs e)
         {
             OpenFileDialog openDialog = new OpenFileDialog();
             openDialog.Filter = TextUtil.FILTER_SKY;
-            openDialog.InitialDirectory = Path.GetDirectoryName(textSkylinePath.Text);
+            try
+            {
+                openDialog.InitialDirectory = Path.GetDirectoryName(textSkylinePath.Text);
+            }
+            catch (ArgumentException)

```

Comment:
Maybe make this Exception. I can imagine this causing exceptions other than ArgumentException, and it seems like we would always just want to fall back to something else or whatever the default is. Maybe improve the comment "pass" to explain better what will happen, e.g. "Accept the default directory"